models/classes/LtiProvider/LtiProviderResource.php

```diff
@@ -0,0 +1,83 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: julien
+ * Date: 29/07/19
+ * Time: 11:47
+ */
+
+namespace oat\taoLti\models\classes\LtiProvider;
+
+use core_kernel_classes_Resource as RdfResource;
+
+class LtiProviderResource extends RdfResource
```

Comment:
this hybrid between LtiProvider and RdfResoruce is not required and can be confusing

Reply:
Removed.